Custom _Input_Fields/code/style.css

```diff
@@ -0,0 +1,199 @@
+* {
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    box-sizing: border-box;
+}
+
+html,
+body {
+    height: 100%;
+}
+
```

Comment:
Target this with class or id

Reply:
I have updated it please review the same!